src/bebb/main.mo

```diff
@@ -236,6 +236,9 @@ actor {
   *
   * @return Either the Entity ID if the call was successful or an error if not
   */
+  let oneMB : Nat = 1048576; // 1 MB
+  private let maxPreviewBlobSize : Nat = 5 * oneMB; 
```

Comment:
We can set this to 2MB for now as this is the IC upload limit for one chunk. Once we have chunking integrated, we could change this.